package.json

```
@@ -11,5 +11,8 @@
   },
   "scripts": {
     "changeset": "changeset"
+  },
+  "dependencies": {
+    "changeset": "^0.2.6"
```

This shouldn't be needed, the `@changesets/cli` dev dependency is the one that is used

```suggestion
  },
  "dependencies": {
    "changeset": "^0.2.6"
```